### ISSUE TITLE
[BUG] Can't run the app after implement it - Android resource linking failed on build

### ISSUE THREAD

**Issue by evanezcent** [tneotia/html-editor-enhanced#21]
**Describe the bug**
I can't run the application after importing the library, even though for configuration I have followed everything in "setup" but it got an error and it can't be run, and it not add the `generated_plugin_registrant.dart` file

**Expected behavior**
It should work well

**Error**
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource linking failed
     [My App path]/my_app_name/build/flutter_inappwebview/intermediates/library_manifest/debug/AndroidManifest.xml:7:5-11:15: AAPT: error: unexpected element <queries> found in <manifest>.


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 33s
The build failed likely due to AndroidX incompatibilities in a plugin. The tool is about to try using Jetifier to solve the incompatibility.
Building plugin flutter_inappwebview...

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':compileReleaseAidl'.
> SDK location not found. Define location with an ANDROID_SDK_ROOT environment variable or by setting the sdk.dir path in your project's local properties file at '/Users/evanezcent/Documents/INSTALLER/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.3.1+1/android/local.properties'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 10s
2

Exception: The plugin flutter_inappwebview could not be built due to the issue above.

```

**Flutter Doctor**
```
[✓] Flutter (Channel stable, 2.0.2, on macOS 11.0.1 20B29 darwin-arm, locale en-ID)
    • Flutter version 2.0.2 at /Users/evanezcent/Documents/INSTALLER/flutter
    • Framework revision 8962f6dc68 (4 weeks ago), 2021-03-11 13:22:20 -0800
    • Engine revision 5d8bf811b3
    • Dart version 2.12.1

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/evanezcent/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] Connected device (1 available)
    • SM A205F (mobile) • RR8M40HTWBD • android-arm64 • Android 10 (API 29)

! Doctor found issues in 1 category.
```

**Comment by tneotia** [comment]
Could you post your `AndroidManifest` code here? Seems the issue is related to that.

**Comment by evanezcent** [comment]
My project made when the flutter version is under 2, but I have upgraded my flutter version and fix it to following the new updates in flutter, is it the problem ?
```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.example.signal_app">
    <!-- io.flutter.app.FlutterApplication is an android.app.Application that
         calls FlutterMain.startInitialization(this); in its onCreate method.
         In most cases you can leave this as-is, but you if you want to provide
         additional functionality it is fine to subclass or reimplement
         FlutterApplication and put your custom class here. -->
    <uses-permission android:name="android.permission.INTERNET"/>
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
    <application
        android:name="io.flutter.app.FlutterApplication"
        android:label="signal_app"
        android:icon="@mipmap/ic_launcher">

        <activity
            android:name=".MainActivity"
            android:launchMode="singleTop"
            android:theme="@style/LaunchTheme"
            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
            android:hardwareAccelerated="true"
            android:windowSoftInputMode="adjustResize">
            <!-- Specifies an Android theme to apply to this Activity as soon as
                 the Android process has started. This theme is visible to the user
                 while the Flutter UI initializes. After that, this theme continues
                 to determine the Window background behind the Flutter UI. -->
            <meta-data
              android:name="io.flutter.embedding.android.NormalTheme"
              android:resource="@style/NormalTheme"
              />
            <!-- Displays an Android View that continues showing the launch screen
                 Drawable until Flutter paints its first frame, then this splash
                 screen fades out. A splash screen is useful to avoid any visual
                 gap between the end of Android's launch screen and the painting of
                 Flutter's first frame. -->
            <meta-data
              android:name="io.flutter.embedding.android.SplashScreenDrawable"
              android:resource="@drawable/launch_background"
              />
            <intent-filter>
                <action android:name="android.intent.action.MAIN"/>
                <category android:name="android.intent.category.LAUNCHER"/>
            </intent-filter>
        </activity>
        <!-- Don't delete the meta-data below.
             This is used by the Flutter tool to generate GeneratedPluginRegistrant.java -->
        <meta-data
            android:name="flutterEmbedding"
            android:value="2" /> 
        
        <!-- FOR HTML EDITOR -->
        <provider
          android:name="com.pichillilorenzo.flutter_inappwebview.InAppWebViewFileProvider"
          android:authorities="${applicationId}.flutter_inappwebview.fileprovider"
          android:exported="false"
          android:grantUriPermissions="true">
          <meta-data
              android:name="android.support.FILE_PROVIDER_PATHS"
              android:resource="@xml/provider_paths" />
        </provider>
    </application>
</manifest>
```

**Comment by tneotia** [comment]
Your AndroidManifest looks good, this seems to be a problem with flutter_inappwebview itself. See https://github.com/pichillilorenzo/flutter_inappwebview/issues/730 and https://github.com/pichillilorenzo/flutter_inappwebview/issues/728. You can try the solution https://github.com/pichillilorenzo/flutter_inappwebview/issues/728#issuecomment-800257978 to see if it fixes it, in the meantime I will update the dependency soon as the plugin author has fixed it in more recent versions.

**Comment by tneotia** [comment]
Fixed via fb16e47b59368965ed0ac4066907fd9133848020, 1.8.0 is now available on Pub with the changes. Thanks for the report!
